src/app/pages/listing/view.html

```diff
@@ -129,16 +129,16 @@ <h2 class="panel-title">Listing Information</h2>
                   <div class="data-label listing__enhanced-label--small">Real World Testing</div>
                 </div>
                 <div ng-if="$ctrl.listing.rwtPlansUrl">
-                  <div class="data-label listing__enhanced-label--small" id="rwt-plans-url">Plans</div>
-                  <div class="read-only-data"><chpl-link-bridge href="$ctrl.listing.rwtPlansUrl" name="rwtPlansUrl"></chpl-link-bridge></div>
+                  <div class="data-label listing__enhanced-label--small">Plans</div>
+                  <div class="read-only-data"><chpl-link-bridge href="$ctrl.listing.rwtPlansUrl" name="rwtPlansUrl" id="rwt-plans-url"></chpl-link-bridge></div>
```

Comment:
These changes are undoing something I already did. Unless you have good reason to, you shouldn't undo something